google-api-client-android/src/main/java/com/google/api/client/googleapis/extensions/android/gms/auth/GoogleAccountCredential.java

```diff
@@ -303,10 +303,16 @@ public void intercept(HttpRequest request) throws IOException {
     @Override
     public boolean handleResponse(
         HttpRequest request, HttpResponse response, boolean supportsRetry) {
-      if (response.getStatusCode() == 401 && !received401) {
-        received401 = true;
-        GoogleAuthUtil.invalidateToken(context, token);
-        return true;
+      try {
+        if (response.getStatusCode() == 401 && !received401) {
+          received401 = true;
+          GoogleAuthUtil.clearToken(context, token);
+          return true;
+        }
+      } catch (GoogleAuthException e) {
+        e.printStackTrace();
```

Comment:
I don't think we can just swallow the exception here.  Why would these exceptions be thrown?

Reply:
Resolved.